xamarin/Xamarin.Forms
# [Android] CarouselView problems updating ObservableCollection

### Description of Change ###

CarouselView binded to a new ObservableCollection filled with Items does not render content.

### Issues Resolved ### 

- fixes #7678

### API Changes ###

 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![7678-before](https://user-images.githubusercontent.com/6755973/65697686-a0823d80-e07b-11e9-8fe6-fdf0ce9d21ba.gif)

#### After
![7678-after](https://user-images.githubusercontent.com/6755973/65697690-a0823d80-e07b-11e9-94e7-242a0fd4e3fb.gif)

### Testing Procedure ###
Launch Core Gallery and search Issue 7678. If the CarouselView is rendering items, works as expected.

### PR Checklist ###
